lib/Mongo/MongoClient.php

```diff
@@ -58,6 +58,22 @@ class MongoClient
     private $client;
 
     /**
+     * @var \MongoDB\Driver\Manager
+     */
+    private $manager;
+
+    private $readPreference = [
```

Comment:
As mentioned in https://github.com/alcaeus/mongo-php-adapter/pull/8#issuecomment-168374248, we could extract these to a trait. At the very least, we should consistently store the same thing in all classes (e.g. all classes store a ReadPreference object or an array).